src/amber/validators/params.cr

```diff
@@ -13,8 +13,14 @@ module Amber::Validators
       @predicate = block
     end
 
-    def apply(params : Amber::Router::Params)
-      raise Exceptions::Validator::InvalidParam.new(@field) unless params.has_key? @field
+    def apply(params : Amber::Router::Params, raise_on_error = true)
```

Comment:
Instead of adding a flag I would recommend to create a RequiredRule class, this way we avoid the `raise_on_error` config argument which I am not fond about. 

See https://gist.github.com/eliasjpr/4d89efe9f2b3cf37d1ad40137fc8b3e5